sound.js

```diff
@@ -54,32 +82,44 @@ Sound.prototype.isLoaded = function() {
   return this._loaded;
 };
 
-Sound.prototype.play = function(onEnd) {
+Sound.prototype.play = function(onEnd, onPlay) {
   if (this._loaded) {
+    onPlay && this.registerOnPlay(onPlay);
```

Comment:
Then... there are two listeners on the one Sound instance?

Reply:
I thought recalling `eventEmitter.addListener` automatically renews the listener. But looks like it doesn't!! I will fix it asap